lib/msal-core/src/ServerRequestParameters.ts

```diff
@@ -36,7 +36,7 @@ export class ServerRequestParameters {
     promptValue: string;
     claimsValue: string;
 
-    queryParameters: string;
+    queryParameters: string|null;
```

Comment:
If strict null checks are on, would we need `|null`?

Reply:
Yes.